### DISCUSSION TITLE
Where to download the OpenSearch Plugin Version

### DISCUSSION THREAD

**Discussion by AESthetix256** [wazuh/wazuh-kibana-app#5269] [Q&A] [answered]
Just as the title states, where would I download the OpenSearch Plugin? All I can see are kibana-plugin downloads, or are these compatible with OpenSearch?

**Comment by Desvelao** [comment, marked as answer]
Hi @AESthetix256 ,

There is no published Wazuh plugin for OpenSearch Dashboards.

The available plugins are for Kibana, and they are not compatibles with OpenSearch Dashboards.

Currently, Wazuh provides a stack with **Wazuh dashboard** and **Wazuh indexer**, both applications are based on distributions of OpenSearch Dashboards and OpenSearch respectively. The **Wazuh dashboard** application has integrated a compatible Wazuh plugin. You can read more about these stack components here:
- https://documentation.wazuh.com/4.3/getting-started/components/index.html
- https://documentation.wazuh.com/4.3/getting-started/components/wazuh-dashboard.html
- https://documentation.wazuh.com/4.3/getting-started/components/wazuh-indexer.html

**Reply by AESthetix256** [reply]
Ahh ok understood.

So what is this issue about exactly: https://github.com/wazuh/wazuh-kibana-app/issues/5263
This was the reason I assumed a plugin for OpenSearch is available, sorry for the misconception.

Thanks for the fast reply!

**Reply by Desvelao** [reply]
> Ahh ok understood.
> 
> So what is this issue about exactly: #5263 This was the reason I assumed a plugin for OpenSearch is available, sorry for the misconception.
> 
> Thanks for the fast reply!

Because the **Wazuh dashboard** application is based on OpenSearch Dashboards, the Wazuh plugin included in the Wazuh dashboard must be compatible with the used version of OpenSearch Dashboard.

The goal of issue #5263 is to check the compatibility of the next Wazuh plugin versions with a specific version of OpenSearch Dashboards, which is newer at the moment. This would allow the creation of a Wazuh dashboard application based on the most recent version of OpenSearch Dashboard including a compatible Wazuh plugin.
